lib/hobo_fields/model/index_spec.rb

```diff
@@ -45,33 +45,23 @@ def each_hash(result)
           end
         end
         connection.indexes(t).map do |i|
-          self.new(model, i.columns, :name => i.name, :unique => i.unique, :where => i.where, :table_name => old_table_name) unless model.ignore_indexes.include?(i.name)
+          self.new(model, i.columns, name: i.name, unique: i.unique, where: i.where, table_name: old_table_name) unless model.ignore_indexes.include?(i.name)
         end.compact
       end
 
       def primary_key?
         name == PRIMARY_KEY_NAME
       end
 
-      def default_name?
-        name == @model.connection.index_name(table, :column => fields)
-      end
-
       def to_add_statement(new_table_name, existing_primary_key = nil)
         if primary_key? && !ActiveRecord::Base.connection.class.name.match?(/SQLite3Adapter/)
           to_add_primary_key_statement(new_table_name, existing_primary_key)
         else
-          r = "add_index :#{new_table_name}, #{fields.map(&:to_sym).inspect}"
-          r += ", :unique => true" if unique
-          r += ", :where => '#{self.where}'" if self.where.present?
-          if default_name?
-            check_name = @model.connection.index_name(self.table, :column => self.fields)
-          else
-            r = "add_index :#{new_table_name}, #{fields.map(&:to_sym).inspect}"
-            r += ", :unique => true" if unique
-            r += ", :name => '#{name}'"
-            r
-          end
```

Comment:
This got much simpler...the above looks like it was actually the victim of bad merge conflict resolution.